test/integration_test/specs/pure-telemetry-cert.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
What is this certificate? It it okay to expose it like this in the code?

Reply:
this is just for testing.

Reply:
Okay, so this is random value and not our pure1 cert?

Reply:
correct